.github/workflows/scripts/android/android-emulator-tests.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift.org open source project
+##
+## Copyright (c) 2025 Apple Inc. and the Swift project authors
+## Licensed under Apache License v2.0 with Runtime Library Exception
+##
+## See https://swift.org/LICENSE.txt for license information
+## See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+##
+##===----------------------------------------------------------------------===##
+
+set -euo pipefail
+
+log() { printf -- "** %s\n" "$*" >&2; }
+error() { printf -- "** ERROR: %s\n" "$*" >&2; }
+fatal() { error "$@"; exit 1; }
+
+ANDROID_PROFILE="Nexus 10"
+ANDROID_EMULATOR_TIMEOUT=300
+
+SWIFTPM_HOME="${XDG_CONFIG_HOME}"/swiftpm
+# e.g., "${SWIFTPM_HOME}"/swift-sdks/swift-DEVELOPMENT-SNAPSHOT-2025-12-11-a_android.artifactbundle/
+SWIFT_ANDROID_SDK_HOME=$(find "${SWIFTPM_HOME}"/swift-sdks -maxdepth 1 -name 'swift-*android.artifactbundle' | tail -n 1)
+
+ANDROID_SDK_TRIPLE="x86_64-unknown-linux-android28"
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --android-sdk-triple=*)
+            ANDROID_SDK_TRIPLE="${1#*=}"
+            shift
+            ;;
+        --android-profile=*)
+            ANDROID_PROFILE="${1#*=}"
+            shift
+            ;;
+        --android-emulator-timeout=*)
+            ANDROID_EMULATOR_TIMEOUT="${1#*=}"
+            shift
+            ;;
+        -*)
+            fatal "Unknown option: $1"
+            ;;
+        *)
+            if [[ -z "$SWIFT_VERSION_INPUT" ]]; then
+                SWIFT_VERSION_INPUT="$1"
+            else
+                fatal "Multiple Swift versions specified: $SWIFT_VERSION_INPUT and $1"
+            fi
+            shift
+            ;;
+    esac
+done
+
+# extract the API level from the end of the triple 
+ANDROID_API="${ANDROID_SDK_TRIPLE/*-unknown-linux-android/}"
+
+# extract the build arch from the beginning of the triple
+ANDROID_EMULATOR_ARCH="${ANDROID_SDK_TRIPLE/-unknown-linux-android*/}"
+
+# x86_64=x86_64, armv7=arm
+ANDROID_EMULATOR_ARCH_TRIPLE="${ANDROID_EMULATOR_ARCH}"
+
```

Comment:
Stick a log message here saying which Android arch is being tested.

Reply:
Added in 15a740d35e7f058cc73ab30a2b80ed251c9054fd